### ISSUE TITLE
InternalError: too much recursion error on webpage

### ISSUE THREAD

**Issue by nuriakman** [mathieustan/vue-datepicker#81]
Hi,

I visit https://vue-datepicker.netlify.app/ to how it works. But there was no datepicket on screen.

I checked to console and I see this error:

InternalError: too much recursion
    isValidLocale vue-datepicker.esm.js:5
    getLocale vue-datepicker.esm.js:5 ------> This line repeats more than 1000 time

**Comment by Aviking88** [comment]
Faced the same and found the cause, It was happening due to locale.

**Comment by yemanjo** [comment]
> Faced the same and found the cause, It was happening due to locale.

Did you find proper workaround?

**Comment by Aviking88** [comment]
yes @yemanjo , I have specified lang at the time of registration,

`Vue.use(VueDatePicker, {
  lang: 'en'
});`

**Comment by Mateis** [comment]
these issues are related: 
- https://github.com/mathieustan/vue-datepicker/issues/84 
- https://github.com/mathieustan/vue-datepicker/issues/94

And there is a hotfix here (for Chrome): https://github.com/mathieustan/vue-datepicker/pull/100
